apps/ledger-live-desktop/src/renderer/components/FirebaseFeatureFlags.tsx

```diff
@@ -23,6 +23,21 @@ const checkFeatureFlagVersion = (feature: Feature) => {
   return feature;
 };
 
+const checkFeatureFlagVersion = (feature: Feature) => {
+  if (
+    feature.enabled &&
+    feature.desktop_version &&
+    !semver.satisfies(__APP_VERSION__, feature.desktop_version)
+  ) {
+    return {
+      enabledOverriddenForCurrentDesktopVersion: true,
+      ...feature,
+      enabled: false,
+    };
+  }
+  return feature;
+};
+
```

Comment:
Duplicate, the same function is just above

Reply:
```suggestion
```

Reply:
Thanks !